### ISSUE TITLE
Summarize Posts shortcode 'before'  argument doesn't work.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [viktorthang/wordpress-custom-content-type-manager#483]
```
What steps will reproduce the problem?
1. Set the before argument in shortcode.

What is the expected output? What do you see instead?
This should allow you to set the <ul> to something more desired (add classes 
etc.)
Instead it is included in the DB query and no results are returned.

What version of the plugin are you using?
Custom Content Types Manager 0.9.7.2-pl

I see in the code that there is a check for the $raw_args['tpl'], but not the 
other format arguments such as $raw_args['before'] $raw_args['after'].

Here is the print_r of $raw_args:

Array
(
    [before] => 
<ul class="nav nav-tabs nav-stacked">
    [post_type] => guideline
    [after] => </ul>
<p class="lead">
)
```

Original issue reported on code.google.com by `SamJClev...@gmail.com` on 3 May 2013 at 7:30

**Comment by GoogleCodeExporter** [comment]
```
I had this working previously by checking if the before and after hashes were 
set and then unsetting them, but this doesn't seem to work anymore.. Not sure 
what is going on now.

Here is my post content:

[summarize-posts before='<ul class="nav nav-tabs nav-stacked">' 
post_type='guideline' after='</ul>']
<li><a href="[+permalink+]">[+post_title+]<i class="icon-chevron-right 
pull-right"></i></a></li>
[/summarize-posts]
```

Original comment by `SamJClev...@gmail.com` on 3 May 2013 at 7:31

**Comment by GoogleCodeExporter** [comment]
```
You may need to flip your quotes -- not sure if WP is smart enough to interpret 
shortcode args wrapped in single-quotes or if they need to be double-quotes.
```

Original comment by `ever...@fireproofsocks.com` on 3 May 2013 at 8:00
- Changed title: **Summarize Posts shortcode 'before'  argument doesn't work.**
- Changed state: **InReview**

**Comment by GoogleCodeExporter** [comment]
```
Committed revision 713117.  Will be released in 0.9.7.4
```

Original comment by `ever...@fireproofsocks.com` on 14 May 2013 at 8:33
- Changed state: **Fixed**
